app/controllers/window/clients/vispdats_controller.rb

```diff
@@ -96,7 +96,8 @@ def upload_file
           visible_in_window: true,
           note: file_params[:note],
           name: file_params[:name],
-          vispdat_id: @vispdat.id
+          vispdat_id: @vispdat.id,
+          consent_form_signed_on: file_params[:consent_form_signed_on]
```

Comment:
Same in this file.  Need to verify the current user has access to the `: consent_form_signed_on` field.

Reply:
there no permission that corresponds to this field - should i create one?

Reply:
My mistake, I forgot we removed `:consent_form_confirmed` from here.  This should be good.